src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -148,7 +148,13 @@ def create_application_gateway(cmd, application_gateway_name, resource_group_nam
                                public_ip_address_type=None, subnet_type=None, validate=False,
                                connection_draining_timeout=0, enable_http2=None, min_capacity=None, zones=None,
                                custom_error_pages=None, firewall_policy=None, max_capacity=None,
-                               user_assigned_identity=None):
+                               user_assigned_identity=None,
+                               enable_private_link=False,
```

Comment:
Could they be updated after the creation? I thought we would have an command group as well.

Reply:
Added